app/templates/views/broadcast/previous-broadcasts.html

```diff
@@ -10,6 +10,11 @@
 {% block maincolumn_content %}
 
   <h1 class="heading-medium">{{ page_title }}</h1>
+  {% if page_title == "Rejected alerts" %}
+      <p class="govuk-body">
+        Alerts that were never broadcast and alerts that can be reworked / edited and then sent for approval
```

Comment:
Possibly rework the text - "Alerts rejected for broadcast that may be edited and resubmitted for approval"